composer.json

```diff
@@ -10,11 +10,12 @@
         "php": "^8.1",
         "google/protobuf": "^3.22",
         "php-http/discovery": "^1.14",
+        "psr/http-client": "^1.0",
         "psr/http-client-implementation": "^1.0",
         "psr/http-factory-implementation": "^1.0",
-        "psr/http-client": "^1.0",
         "psr/http-message": "^1.0.1|^2.0",
         "psr/log": "^1.1|^2.0|^3.0",
+        "ramsey/uuid": "^4.3",
```

Comment:
I think we can support all the way back to `3.0.0`: https://github.com/ramsey/uuid/blob/3.0.0/src/Uuid.php#L647